src/features/post/components/StepFunnel/step1.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react';
+import * as s from './style.css';
+import TypeCard from '../TypeCard';
+import type { PostType } from '@/libs/types/post';
+
+const Step1 = () => {
+  const [selectedTypes, setSelectedTypes] = useState<PostType[]>([]);
+
+  const handleSelectType = (type: PostType) => {
+    setSelectedTypes(prev => (prev.includes(type) ? prev.filter(t => t !== type) : [...prev, type]));
+  };
+
+  return (
+    <div className={s.Wrapper}>
+      <h1 className={s.Head}>거래하실 종류를 선택해 주세요</h1>
+      <div className={s.Container}>
+        <TypeCard types="대여" isSelected={selectedTypes.includes('RENTAL')} onClick={() => handleSelectType('RENTAL')} />
+        <TypeCard types="판매" isSelected={selectedTypes.includes('SALE')} onClick={() => handleSelectType('SALE')} />
```

Comment:
```suggestion
        <TypeCard types="RENTAL" isSelected={selectedTypes.includes('RENTAL')} onClick={() => handleSelectType('RENTAL')} />
        <TypeCard types="SALE" isSelected={selectedTypes.includes('SALE')} onClick={() => handleSelectType('SALE')} />
```
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 헤어나올수 없는 타입에러의 늪

Reply:
찐찐찐찐막 이거 해결하고 토큰도 넣었습니다